doc/pr2_tutorials/planning/launch/move_group_interface_tutorial.launch

```diff
@@ -1,7 +1,5 @@
 <launch>
 
-  <include file="$(find pr2_moveit_config)/launch/demo.launch"/>
-
```

Comment:
I agree it should be consistent across the tutorials, but I don't have time to work on the other tutorials right now. This one alone took an entire 8 hours.

I really dislike the workflow of having a single roslaunch file that includes Rviz and whatever demo/script you are testing. I way prefer to launch Rviz and move it to the area on my screen I want, then be able to launch and relaunch the scripts that actually do the things I'm testing. Previously this tutorial had a 20 second sleep "to allow Rviz to launch" which I really detest - its typically a huge waste of time for the user and its confusing.